clients/src/main/java/org/apache/kafka/common/requests/GroupCoordinatorRequest.java

```diff
@@ -16,32 +16,58 @@
 import org.apache.kafka.common.protocol.ApiKeys;
 import org.apache.kafka.common.protocol.Errors;
 import org.apache.kafka.common.protocol.ProtoUtils;
-import org.apache.kafka.common.protocol.types.Schema;
 import org.apache.kafka.common.protocol.types.Struct;
 
 import java.nio.ByteBuffer;
 
 public class GroupCoordinatorRequest extends AbstractRequest {
-    
-    private static final Schema CURRENT_SCHEMA = ProtoUtils.currentRequestSchema(ApiKeys.GROUP_COORDINATOR.id);
     private static final String GROUP_ID_KEY_NAME = "group_id";
 
-    private final String groupId;
+    public static class Builder extends AbstractRequest.Builder<GroupCoordinatorRequest> {
+        private String groupId;
+
+        public Builder(String groupId) {
+            super(ApiKeys.GROUP_COORDINATOR);
+            this.groupId = groupId;
+        }
 
-    public GroupCoordinatorRequest(String groupId) {
-        super(new Struct(CURRENT_SCHEMA));
+        public Builder setGroupId(String groupId) {
+            this.groupId = groupId;
+            return this;
+        }
+
+        @Override
+        public GroupCoordinatorRequest build() {
+            short version = version();
+            return new GroupCoordinatorRequest(this.groupId, version);
+        }
+
+        @Override
+        public String toString() {
+            StringBuilder bld = new StringBuilder();
+            bld.append("{type: GroupCoordinatorRequest, groupId: ");
```

Comment:
Braces seem a little more familiar to me because of JSON.  But if parentheses are the way to go, I can change it.  It's a big change though since all the builders have a toString.  Are we sure we want the parens?